ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -314,6 +319,24 @@ func createCluster(context *cli.Context, awsClients *AWSClients, commandConfig *
 	}
 
 	if launchType == config.LaunchTypeEC2 {
+		instanceType, err := populateInstanceTypeParameter(cfnParams)
+		if err != nil {
+			return err
+		}
+		supportedInstanceTypes, err := awsClients.EC2Client.DescribeInstanceTypeOfferings(commandConfig.Region())
+		if err != nil {
+			return errors.Wrapf(err, "No instance type found in region %s", commandConfig.Region())
```

Comment:
The error returns by `DescribeInstanceTypeOfferings` might not only because "no instance type found"? It could be other reason. And also I guess "No instance type found in region %s" is already wrapped up in `DescribeInstanceTypeOfferings`?

Reply:
I copied this exactly as it was previously, but I'll update it to wrap with "failed to describe instance type offerings: %w". I think this is what you're suggesting?

Reply:
yes that's my bad. thanks for fixing that!

Reply:
maybe just "describe instance type offerings: %w" would be enough.